src/Symfony/Bundle/FrameworkBundle/FrameworkBundle.php

```diff
@@ -34,8 +34,8 @@
 use Symfony\Bundle\FrameworkBundle\DependencyInjection\Compiler\TranslationDumperPass;
 use Symfony\Bundle\FrameworkBundle\DependencyInjection\Compiler\SerializerPass;
 use Symfony\Bundle\FrameworkBundle\DependencyInjection\Compiler\UnusedTagsPass;
-use Symfony\Bundle\FrameworkBundle\DependencyInjection\Compiler\ConfigCachePass;
 use Symfony\Bundle\FrameworkBundle\DependencyInjection\Compiler\ValidateWorkflowsPass;
+use Symfony\Component\Config\DependencyInjection\Compiler\ConfigCachePass;
```

Comment:
Should be `Symfony\Component\Config\DependencyInjection\ConfigCachePass`, right?